### PR TITLE
🗞️ Adding in Default Colors to print outs

### PR DESCRIPTION
**Pull request by sirarthurmoney**
### In thi PR

- Adding in default colors

Screenshots:
 
Screenshot 2024-02-09 at 11 10 14 AM
Screenshot 2024-02-09 at 11 13 39 AM
Screenshot 2024-02-09 at 11 14 03 AM
Screenshot 2024-02-09 at 11 14 44 AM

### CODE REVIEW

File: packages/io-devtools/src/stdio/printer.ts [hunk 1]
```diff
@@ -63,20 +86,39 @@ export const printRecord = <TRecord extends object>(obj: TRecord, title?: string
  * the property name.
  *
  * @param {Record<string | number, unknown>} records
- * @param {string[]} [header]
+ * @param {string[]} [header] - header row if provided (OPTIONAL)
+ * @param center - center text if true (OPTIONAL)
+ * @param keyColor - color of keys in table (DEFAULT: WHITE)
+ * @param valueColor - color of values in table (DEFAULT: MAGENTA)
```
Comment (janjakubnanista) [inline packages/io-devtools/src/stdio/printer.ts:92]: Here the JSDoc should be something like:

```typescript
/**
 * @param {boolean} [center] ...
 * @param {Preprocess} [keyColor] ...
 */
```

The `[]` mean a param is optional and the `{}` denote the type

File: packages/io-devtools/src/stdio/printer.ts [hunk 2]
```diff
@@ -110,9 +152,17 @@ export const printCrossTable = <TRecord extends Record<string | number, unknown>
  * Helper utility for printing out boolean values
  *
  * @param {boolean | null | undefined} value
+ * @param nullColor
+ * @param trueColor
+ * @param falseColor
```
Comment (janjakubnanista) [inline packages/io-devtools/src/stdio/printer.ts:157]: The same as above

File: packages/ua-devtools-evm-hardhat/src/tasks/oapp/peers.get.ts [hunk 3]
```diff
@@ -67,10 +67,11 @@ const action: ActionType<TaskArgs> = async ({ oappConfig: oappConfigPath, logLev
...
         console.log(
-            printCrossTable(peerNetworkMatrix, ['from → to', ...points.map(({ networkName }) => networkName)]),
-            `\n\t${printBoolean(true)} - Connected\n`,
-            `\t${printBoolean(false)} - Not Connected\n`,
-            `\t${printBoolean(undefined)} - Ignored`
+            printCrossTable(peerNetworkMatrix, ['from → to', ...points.map(({ networkName }) => networkName)], true) +
+                '\n',
+            `${printBoolean(true)} - Connected\n`,
+            `${printBoolean(false)} - Not Connected\n`,
+            `${printBoolean(undefined)} - Ignored`
```
Comment (janjakubnanista) [inline packages/ua-devtools-evm-hardhat/src/tasks/oapp/peers.get.ts:74]: Here you can also split it into several logs:

```typescript
console.log(printCrossTable(peerNetworkMatrix, ['from → to', ...points.map(({ networkName }) => networkName)], true))
console.log(`${printBoolean(true)} - Connected`)
console.log(`${printBoolean(false)} - Not Connected`)
console.log(`${printBoolean(undefined)} - Ignored `)
```

**Review by janjakubnanista — CHANGES_REQUESTED** [review]

File: packages/io-devtools/src/stdio/printer.ts [hunk 4]
```diff
@@ -63,20 +86,39 @@ export const printRecord = <TRecord extends object>(obj: TRecord, title?: string
...
+        // Keep property column 'left'
+        const colAligns: HorizontalAlignment[] = ['left']
+        // Center-align for all other columns
+        for (let i = 0; i < records.length; i++) {
+            colAligns.push('center')
+        }
+        // Set the alignment for the table
+        table.options.colAligns = colAligns
+    }
+
```
Comment (janjakubnanista) [inline packages/io-devtools/src/stdio/printer.ts:121]: You can also do

```typescript
const colAligns = center 
  ? ['left', ...Array.from({ length: records.length }).fill('center')]
  : undefined
```

Or shorter:

```typescript
const colAligns = center 
  ? ['left', ...records.slice().fill('center')]
  : undefined
```

**Review by janjakubnanista — CHANGES_REQUESTED** [review]

File: packages/io-devtools/src/stdio/printer.ts (hunk 4 above)
Comment (sirarthurmoney) [reply packages/io-devtools/src/stdio/printer.ts:121]: Yes there are better👍

**Review by janjakubnanista — APPROVED** [review]
📅🌮

**Pull request merged**